### PR TITLE
Master client merges for two files from zVM

### PR DESCRIPTION
**Pull request by chuckbrazie**
These are the only two updates I found in zVM xCAT xCAT-client which need to be merged. Let me know if the merge I did is correct.

### CODE REVIEW

**Comment by daniceexi** [comment]
Hi @immarvin, the changes in this pull request come from the zVM team. Could you review the changes to make sure they are OK for general xCAT functions?

File: xCAT-client/bin/xdsh [hunk 1]
```diff
@@ -573,9 +575,11 @@ sub parse_args_xdcp
             'F|File=s'              => \$options{'File'},
             'h|help'                => \$options{'help'},
             'i|rootimg=s'           => \$options{'rootimg'},
-            'l|user=s'              => \$options{'user'},
+                    'ip=s'             => \$options{'ip'},
```
Comment (immarvin) [inline xCAT-client/bin/xdsh]: please indent the modified lines with the other lines, thanks

File: xCAT-client/bin/xdsh [hunk 2]
```diff
@@ -573,9 +575,11 @@ sub parse_args_xdcp
...
+                    'l|user=s'              => \$options{'user'},
```
Comment (immarvin) [inline xCAT-client/bin/xdsh]: please indent the modified lines with the other lines, thanks

File: xCAT-client/bin/xdsh [hunk 3]
```diff
@@ -573,9 +575,11 @@ sub parse_args_xdcp
...
             'm|monitor'             => \$options{'monitor'},
             'o|node-options=s'      => \$options{'node-options'},
+                    'show=s'           => \$options{'show'},
```
Comment (immarvin) [inline xCAT-client/bin/xdsh]: please indent the modified lines with the other lines, thanks

File: xCAT-client/bin/genimage [hunk 4]
```diff
@@ -57,13 +57,19 @@ sub print_usage
     print "Usage:\n";
     print "    genimage\n\n";
     print "    genimage --dryrun\n\n";
+    print '    genimage  -o <osver> [-a <arch>] -p <profile> -i <nodebootif> -n <nodenetdrivers> [--onlyinitrd] [-r <otherifaces>] [-k <kernelver>] [-g <krpmver>] [-m statelite] [-l rootlimitsize] [-t tmplimitsize] [--permission <permission>] [--interactive] [--dryrun]'."\n\n";
```
Comment (immarvin) [inline xCAT-client/bin/genimage]: is this line duplicate with the line #61?

Comment (chuckbrazie) [inline xCAT-client/bin/genimage]: It is actually a "different" line where some of the keys are optional instead of required.

**Comment by immarvin** [comment]
agree to merge

**Pull request merged**